### ISSUE TITLE
Application Insights .NET SDK does not respect LogLevel.None

### ISSUE THREAD

**Issue by vmcbaptista** [microsoft/ApplicationInsights-dotnet#2666]
Repro steps:
Create an ASP.NET Core application with Application Insights configured.

Add a log like this: `_logger.Log(LogLevel.None, "Do not show")`

Actual behavior:
* Log is sent to application insights and is visible in the query editor with severitylevel = 0

Expected behavior:
* Log is not sent to application insights

**Comment by cijothomas** [comment]
Known issue/feature/bug/capability!

But I do not know why ilogger even sends the LogLevel.None to the providers (like ApplicationInsights!).

**Comment by vmcbaptista** [comment]
> Known issue/feature/bug/capability!
> 
> But I do not know why ilogger even sends the LogLevel.None to the providers (like ApplicationInsights!).

Yeah, to be fair I never used this loglevel, but I'm depending on a library this do use it and I'm getting a lot of unwanted messages in AI 😟

Is this something you plan to fix at AI SDK, or shall we rather raise this to the .NET logging team?

**Comment by cijothomas** [comment]
Since all other providers are doing this check, probably makes sense for application insights to do this as well.

**Comment by vmcbaptista** [comment]
Cool. Would be possible to have any ETA about when this could be fixed and released? Is this something I cal help with  PR?

**Comment by cijothomas** [comment]
No ETA. This is not a high priority item.

Happy to accept a PR if you can send one! it should be straightforward.
https://github.com/microsoft/ApplicationInsights-dotnet/blob/main/LOGGING/src/ILogger/ApplicationInsightsLogger.cs#L68 This is where we need change.

This is a reference https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLogger.cs#L70
